modules/xfem/include/base/XFEM.h

```diff
@@ -228,8 +245,9 @@ class XFEM : public XFEMInterface
 
   std::map<unique_id_type, XFEMCutElem *> _cut_elem_map;
   std::set<const Elem *> _crack_tip_elems;
-  ElementPairLocator::ElementPairList _sibling_elems;
-  ElementPairLocator::ElementPairList _sibling_displaced_elems;
+  std::set<const Elem *> _crack_tip_elems_need_heal;
```

Comment:
I'd rename this to `_crack_tip_elems_to_be_healed`.  Also, add one-line doxygen comments for these variables.